homeassistant/components/motionblinds_ble/entity.py

```diff
@@ -39,6 +40,7 @@ def __init__(
         self.device = device
         self.entry = entry
         self.entity_description = entity_description
+        self.entity_registry_enabled_default = entity_registry_enabled_default
```

Comment:
This is not needed for this to work

Reply:
Just the field in the entity_description is enough

Reply:
Oh interesting. How does the entity description work then? Is it just a container for all of these values that is used to populate the attributes like `self.entity_registry_enabled_default`?